doc/nrf/device_guides/working_with_nrf/nrf70/developing/sap.rst

```diff
@@ -0,0 +1,118 @@
+.. _nRF70_soft_ap_mode:
+
+Soft AP mode
+############
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The nRF70 Series device supports Software-enabled Access Point ( :abbr:`SAP (Software-enabled Access Point)` ) mode.
+This mode allows a device to act as a virtual router or temporary access point (AP) for other station devices to connect to it.
+This is typically used in scenarios where you want to share the device's internet connection with other station devices.
+For example, when you set up a hotspot on your smartphone, it's using SAP mode.
+The smartphone acts as a virtual router, allowing other station devices to connect to the internet through it.
+
+The nRF70 Series devices use SAP mode to support Wi-Fi based provisioning.
+The SAP mode support is experimental and is not recommended for production use.
```

Comment:
Do we need to state this here ? I guess this should be updated in https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-13562/nrf/releases_and_maturity/software_maturity.html#wi-fi-feature-support ?

Reply:
To avoid redundancy, I suggest at least mentioning "The SAP mode support is experimental" and linking to software_maturity.rst under "experimental", as we do in other places.

Reply:
ok, I forgot to tag this support as `EXPERIMENTAL` will do it in this PR.